dev/scripts/exglobal_diag.sh

```diff
@@ -150,8 +150,7 @@ for loop in ${loops}; do
     while [[ ${n} -lt ${ntype} ]]; do
         n=$((n + 1))
         for type in ${diagtype[n]}; do
-            #shellcheck disable=SC2012,SC2312
-            count=$(ls dir.*/"${type}_${loop}"* 2> /dev/null | wc -l)
+            count=$(find ./ -path "./dir.*/${type}_${loop}*" -type f -printf "." | wc -c)
```

Comment:
Find directly and echo 0 if nothing is found (faster).
```suggestion
            count=$(find ./dir.*/${type}_${loop}* -type f -printf "." | wc -c || echo 0)
```